`dryad stem files` crash

Running `dryad stem files` gave the following crash:
```
panic: interface conversion: interface {} is nil, not string

goroutine 1 [running]:
main._buildCLI.func41({{0x584800, 0xc000076540}, {0xc0001da1e0, 0x3, 0x3}, {0x0, 0x0, 0x0}, 0xc0001da210})
	/home/socks/work/dryad/dyd/heap/stems/blake2b-8cea3e1ab35358d554638de47c7111f9/dyd/stems/src/dyd/assets/main.go:1554 +0x1b9
dryad/cli.(*app).Run(0xc000076540, {0xc000016180?, 0x0?, 0x0?}, {0x5834b8, 0xc000014018})
	/home/socks/work/dryad/dyd/heap/stems/blake2b-8cea3e1ab35358d554638de47c7111f9/dyd/stems/src/dyd/assets/cli/app.go:204 +0x34c
main.main()
	/home/socks/work/dryad/dyd/heap/stems/blake2b-8cea3e1ab35358d554638de47c7111f9/dyd/stems/src/dyd/assets/main.go:1715 +0x83

```

--- a/docs/usage/cli/dryad-stem-files.md
+++ b/docs/usage/cli/dryad-stem-files.md
@@ -9,10 +9,13 @@ grand_parent: Usage
 
 ```
 $ dryad stem files --help
-dryad stem files [--exclude=string]
+dryad stem files [--exclude=string] [path]
 
 Description:
     list the files in a stem
+
+Arguments:
+    path            path to the stem base dir, optional
 
 Options:
         --exclude   a regular expression to exclude files from the list. the regexp matches against the file path relative to the stem base directory

--- a/dyd/roots/dryad/src/dyd/assets/main.go
+++ b/dyd/roots/dryad/src/dyd/assets/main.go
@@ -1542,15 +1542,18 @@ func _buildCLI() cli.App {
 		})
 
 	var stemFiles = cli.NewCommand("files", "list the files in a stem").
-		// WithArg(cli.NewArg("path", "path to the stem base dir")).
+		WithArg(cli.NewArg("path", "path to the stem base dir").AsOptional()).
 		WithOption(cli.NewOption("exclude", "a regular expression to exclude files from the list. the regexp matches against the file path relative to the stem base directory")).
 		WithAction(func(req cli.ActionRequest) int {
+			var args = req.Args
 			var options = req.Opts
 
 			var err error
 			var matchExclude *regexp.Regexp
 
-			if options["exclude"] != "" {
+			fmt.Println("stemFiles options", options, options["exclude"])
+
+			if options["exclude"] != nil && options["exclude"] != "" {
 				matchExclude, err = regexp.Compile(options["exclude"].(string))
 				if err != nil {
 					log.Fatal(err)
@@ -1558,10 +1561,20 @@ func _buildCLI() cli.App {
 			}
 
 			var path string
-			path, err = os.Getwd()
-			if err != nil {
-				log.Fatal(err)
+			if len(args) > 0 {
+				path = args[0]
+				path, err = filepath.Abs(path)
+				if err != nil {
+					log.Fatal(err)
+				}
 			}
+			if path == "" {
+				path, err = os.Getwd()
+				if err != nil {
+					log.Fatal(err)
+				}
+			}
+
 			err = dryad.StemFiles(
 				dryad.StemFilesArgs{
 					BasePath:  path,

--- a/dyd/roots/dryad/src/dyd/assets/main.go
+++ b/dyd/roots/dryad/src/dyd/assets/main.go
@@ -1551,8 +1551,6 @@ func _buildCLI() cli.App {
 			var err error
 			var matchExclude *regexp.Regexp
 
-			fmt.Println("stemFiles options", options, options["exclude"])
-
 			if options["exclude"] != nil && options["exclude"] != "" {
 				matchExclude, err = regexp.Compile(options["exclude"].(string))
 				if err != nil {

